public/javascripts/app.js

```diff
@@ -1,64 +1,69 @@
+            $rootScope.$apply();
+        });
+
+        $rootScope.$on('$stateChangeStart', function(event, toState){
+            // Would print "Hello World!" when 'parent' is activated
+            // Would print "Hello UI-Router!" when 'parent.child' is activated
+        })
     })
-  })
 
-  .config(function($stateProvider, $urlRouterProvider, $sceProvider){
-    // Disable Sce 
-    $sceProvider.enabled(false);
-    //
-    // For any unmatched url, redirect to /state1
-    $urlRouterProvider.otherwise("/search");
-    //
-    // Now set up the states
-    $stateProvider
-      .state('search', {
-        url: "/search",
-        templateUrl: "views/search",
-        controller: 'searchCtrl',
-        resolve: {
-          types: model.getTypes
-        }
-      })
-      .state('admin', {
-        abstract: true,
-        url: "/admin",
-        template: '<ui-view/>'
-      })
-      .state('admin.overview', {
-        url: "",
-        templateUrl: "views/admin",
-        controller: 'adminCtrl',
-        resolve: {
-          // modules: model.getAvailableModules,
-          types: model.getTypes
-        }
-      })
-      .state('admin.mapping', {
-        url: "/mapping/{type}",
-        templateUrl: "views/mapping",
-        controller: 'mappingCtrl',
-        parent: 'admin',
-        resolve: {
-          mime_types: model.getAvailableModules,
-          types: model.getTypes,
-          mapping: model.getMapping
-        }
-      });
-  });
+    .config(function($stateProvider, $urlRouterProvider, $sceProvider){
+        // Disable Sce
+        $sceProvider.enabled(false);
+        //
+        // For any unmatched url, redirect to /state1
+        $urlRouterProvider.otherwise("/search");
+        //
+        // Now set up the states
+        $stateProvider
+            .state('search', {
+                url: "/search",
+                templateUrl: "views/search",
+                controller: 'searchCtrl',
+                resolve: {
+                    types: model.getTypes
+                }
+            })
+            .state('admin', {
+                abstract: true,
+                url: "/admin",
+                template: '<ui-view/>'
+            })
+            .state('admin.overview', {
+                url: "",
+                templateUrl: "views/admin",
+                controller: 'adminCtrl',
+                resolve: {
+                    // modules: model.getAvailableModules,
+                    types: model.getTypes
+                }
+            })
+            .state('admin.mapping', {
+                url: "/mapping/{type}",
+                templateUrl: "views/mapping",
+                controller: 'mappingCtrl',
+                parent: 'admin',
+                resolve: {
+                    mime_types: model.getAvailableModules,
+                    types: model.getTypes,
+                    mapping: model.getMapping
+                }
+            })
+            .state('user/overview', {
```

Comment:
States aren't written with slashes (/) as separator, but with dots (.). Since this is an overview, you could simply use `.state('user', {` for it.